nest/00. Foreword.md

```diff
@@ -0,0 +1,21 @@
+# Getting started with Nest
+## Foreword
+Welcome to Nest's official getting-started guide! This guide assumes you are using the latest version of Nest, which is 0.1.0 at the time of writing.
+
+In order to avoid any confusion, we will refer to the project as a whole as `Nest` (with an upper case n), while we will refer to the binary as `nest` (with a lower case n). Thus, `finest` refer to its eponymous binary, and is included in Nest.
+
+[//]: # (TODO: add link to the installation instructions page of Raven-OS)
+Nest is a dependency-based package manager for [Raven-OS](https://raven-os.org), which is an adaptive, customizable and evolutive open-source Operating System (OS) based on the Linux kernel. To use Nest, it's recommended to install Raven-OS and to run Nest in it, because Nest may overlap with an already installed package manager. To install Raven-OS, follow the instructions [here](). Nest is included in the installation of Raven-OS by default, so you can start using it right away.
+
+## How to read this guide
+nest is the binary targeted towards newcomers. If this is the first time you are using Raven-OS, or more generally a GNU/Linux distribution, it's strongly recommended that you use this binary instead of finest. The latter is targeted towards more experienced users, and has more advanced commands. Both are installed by default with Nest, so you don't need to worry about installing them separately.
+
+[//]: # (TODO: Add link to the chapter 1)
+[//]: # (TODO: Add link to the chapter 2)
+[//]: # (TODO: Add link to the chapter 3)
+[//]: # (TODO: Add link to the chapter 4)
+The [introduction]() details general but important concepts to understand to be able to use Nest. The [chapter 2]() explains nest's commands. The [chapter 3]() explains some important concepts behind Nest. Although reading it is recommended, it's not required for the usage of nest (but it is for the usage of finest). The [chapter 4]() details the key concepts behind finest and its commands. In the [chapter 5](), you will find a cheat sheet, with all nest's and finest's commands, and a short explanation on what they do. Finally, to conclude this guide, you will find a glossary in the [chapter 6](). It links all the commands, the important concepts and the technical terms mentioned in this guide to their respective chapters. If you're searching for explanations related to Nest, you should search there.
```

Comment:
Here are some suggestion to make some sentences more straightforward and to-the-point, with less repetitions (and a bit less frenglish):

```suggestion
The [introduction]() details generic but important concepts to grasp before being able to use Nest. The [chapter 2]() explains nest's commands while the [chapter 3]() explains some important concepts behind Nest. Although reading it is recommended, it is not required for a daily usage of nest (but it is for the usage of finest). The [chapter 4]() details the key concepts behind finest and its commands. In the [chapter 5](), you will find a cheat-sheet, with all nest's and finest's commands, and a short explanation on what they do. Finally, to conclude this guide, you will find a glossary in the [chapter 6](). It links all the commands, the important concepts and the technical terms mentioned in this guide to their respective chapters. If you're searching for explanations related to Nest, you should search there.
```

Reply:
"generic but important concepts" ? I don't think "generic" is the word you're looking for.

Reply:
Why not?

Reply:
`general`has the idea of being like overall concepts of Nest, without going too much into detail
whereas `generic` sounds like the concepts are related to a whole group or class, when it's only related to Nest. (Well ok there are `finest` and `nest`, but it feels weird to say `generic` instead of `general` for me)

Reply:
Ok! I'm fine with that reasoning! :)